task-maker-format/src/ioi/finish_ui.rs

```diff
@@ -389,12 +404,25 @@ impl FinishUI {
     }
 
     /// Print the score fraction of a solution using colors.
-    fn print_score_frac(&mut self, score: f64, max_score: f64) {
+    fn print_score_frac(&mut self, score: f64, max_score: f64, score_precision: Option<u8>) {
+        let score_precision = score_precision.unwrap_or(2) as usize;
```

Comment:
Why do we want two decimal places by default in the subtask's score?

Reply:
Good point, I just replicated what we had, but your idea of score_precision + log10(max_score) makes sense.